### PR TITLE
feat: add user api token logs

### PR DESCRIPTION
**Pull request by voikin**
more details in `asp-715`

### CODE REVIEW

File: controllers/user_api_tokens.go [hunk 1]
```diff
@@ -79,6 +80,11 @@ func (c *ApiController) AddApiToken() {
 		return
 	}
 
+	token := tokenUser.AccessKey + tokenUser.AccessSecret
+	isRootUser := currentUser.Owner == "built-in" && currentUser.Name == "admin"
```
Comment (svgrafov) [inline controllers/user_api_tokens.go:84]: Please add constants for "built-in" and "admin" in this context, or a helper function

Comment (voikin) [reply controllers/user_api_tokens.go]: done

Comment (svgrafov) [reply controllers/user_api_tokens.go]: There's one more case in this PR where it is not yet done

Comment (voikin) [reply controllers/user_api_tokens.go]: sorry, I missed this one, now it's done)

**Review by svgrafov — APPROVED** [review]

**Review by svgrafov — APPROVED** [review]

**Pull request merged**